skeleton/notification/type/demo.php.twig

```diff
@@ -157,9 +157,11 @@ class {{ EXTENSION.extension_name }} extends \phpbb\notification\type\base
 	 *
 	 * @param array $data The type specific data
 	 * @param array $pre_create_data Data from pre_create_insert_array()
+{% if REQUIREMENTS.phpbb_version_min matches '/^[\\D]*3\\.1.*$/' %}
```

Comment:
Such IF statement should be removed.

Reply:
I feel like they can be left in place, eventhough 3.1 is EOL. People are still allowed to develop extensions for them. Might aswell provide the correct syntax for them.
Only thing that should be considered is bumping the 'default minimum requirement' to 3.2.0, so if users want to develop for 3.1.* they have to actively change it, to prevent users being lazy and ending up with the wrong examples.

Reply:
We should not remove 3.1 yet as I'm sure people may still want to develop extension's for 3.1.

But @mrgoldy is right that we should change out the default values for min and max phpBB versions shown on the skeleton from... Min. should now be `>= 3.2.0` and max. should be `< 3.3.0@dev` but that's for another PR.